workflow-runtime/src/main/java/com/squareup/workflow1/internal/WorkflowNode.kt

```diff
@@ -100,7 +101,9 @@ internal class WorkflowNode<PropsT, StateT, OutputT, RenderingT>(
     workflow: StatefulWorkflow<PropsT, *, OutputT, RenderingT>,
     input: PropsT
   ): RenderingT =
-    renderWithStateType(workflow as StatefulWorkflow<PropsT, StateT, OutputT, RenderingT>, input)
+    Tracing.traceId({ identifier }) {
+      renderWithStateType(workflow as StatefulWorkflow<PropsT, StateT, OutputT, RenderingT>, input)
+    }
```

Comment:
This is tracing we add for any and all Workflow Nodes that we care about measuring the time for their render.

Reply:
You sure we can't accomplish this with the existing interceptor API? This seems like a very big new back door addressing the same kind of use case. It's not clear to me that this is any more expressive.

Reply:
We likely could reproduce something close the use for each `WorkflowNode` but the other example - that of the internal `WorkflowRunner` "Render Pass" timing wouldn't be possible I don't think. I discussed this with @RBusarow but maybe we should sync up again briefly to discuss it.